packages/auth/src/sandbox-auth-repo.ts

```diff
@@ -1,32 +1,37 @@
 import type { AuthRepo } from './auth-repo';
 import type { User } from './user';
-import { InvariantError, NoAuthenticatedUserError } from './errors';
+import { NoAuthenticatedUserError } from './errors';
 import { invariant, isProduction, warning } from './utils';
 
 export type SandboxAuthOptions =
   | boolean
   | {
       enabled?: boolean;
       user: User | null | (() => User | null) | (() => Promise<User | null>);
+      autologin?: boolean;
```

Comment:
I got your idea, but let's keep it simpler. The developer has to `requestLogin` to authenticate with the sandbox user.

```suggestion
```